### PR TITLE
Fixed the Pokedex Filtering for Type Search

### PR DESCRIPTION
**Pull request by Ash031**

### CODE REVIEW

**Comment by Ishadijcks** [comment]
The second filter doesn't seem to work correctly.

If I select `Normal/Flying` I only want pokemons with that exact type

File: src/scripts/wildBattle/PokemonsPerRoute.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ const pokemonsPerRoute = {
             headbutt: [],
         },
         5: {
-            land: ['Pidgey', 'Pidgeotto', 'Rattata', 'Jigglypuff', 'Oddish', 'Meowth', 'Mankey', 'Abra', 'Bellsprout'],
+            land: ['Pidgey', 'Pidgeotto', 'Rattata', 'Jigglypuff', 'Oddish', '!(((owth', 'Mankey', 'Abra', 'Bellsprout'],
```
Comment (Ishadijcks) [inline src/scripts/wildBattle/PokemonsPerRoute.ts]: ?

Comment (Ash031) [reply src/scripts/wildBattle/PokemonsPerRoute.ts]: Probably hit the wrong stuff when editing the parenthesis as I was browsing throught the repo, will fix.

Comment (Ishadijcks) [reply src/scripts/wildBattle/PokemonsPerRoute.ts]: Good :)

Comment (Ash031) [reply src/scripts/wildBattle/PokemonsPerRoute.ts]: It's fixed

**Comment by Ishadijcks** [comment]
Why are there things changed in the `package-lock`? Did you run `npm update`? Please remove those

**Comment by Ishadijcks** [comment]
Otherwise the logic seems to work correctly :)

**Comment by Ash031** [comment]
> Why are there things changed in the `package-lock`? Did you run `npm update`? Please remove those

I haven't run anything other than npm install and npm start, probably some minor automatic changes from the package manager?

**Comment by Ishadijcks** [comment]
Hmm odd, that shouldn't happen I think

**Pull request merged**